Makefile

```diff
@@ -29,7 +31,10 @@ all: build
 
 .PHONY: build
 build:
-	@go run ./internal/cmd/dockerbuild -org "$(DOCKER_ORG)" -- $(DOCKER_BUILD_EXTRA_ARGS)
+	docker buildx inspect "$(DOCKER_BUILDER)" 2> /dev/null || docker buildx create --use --bootstrap --name="$(DOCKER_BUILDER)"
```

Comment:
Added this so local builds behave the same as ones in CI (they'll both use the `docker-container` driver).